core/types/block.go

```diff
@@ -378,6 +426,17 @@ func (b *Block) WithBody(transactions []*Transaction, uncles []*Header) *Block {
 	return block
 }
 
+// WithBody2 returns a new block with the given transaction, uncle, and
+// withdrawal contents.
+func (b *Block) WithBody2(transactions []*Transaction, uncles []*Header, withdrawals []*Withdrawal) *Block {
```

Comment:
Ah my eyes! 
Please use language.

Reply:
cc @fjl - he recommend this naming convention.